src/components/RestaurantCard/restaurantCard.module.scss

```diff
@@ -0,0 +1,49 @@
+@import './../../constants/fonts.scss';
+
+
+.cardTitle {
+  @include mobile-h1;
+  font-weight: 400;
+  letter-spacing: 2.67px; 
+  text-align: left;
+
+  @media (min-width: 1440px) {
+    @include desktop-h1;
```

Comment:
if you have many placed with h1 and font weight 400 you can do 
mobile-h1-thin (400)
mobile-h1-bold